content/archive/july2024.md

```diff
@@ -0,0 +1,23 @@
+### What are the beekeepers up to?
+
+We're settling into our caretaking role and feeling good. Days are long and the garden is popping with color. We have a stand of popcorn that's taller than Adrienne (not taller than Justin quite yet) with the tassles out and silks starting to form, and a stand of scarlet runner beans that seem like they're growing about six inches a day (maybe a little bit of an exaggeration, but not much). Tomatoes will be here before we know it, as will the fruit harvest - there are about 100 fruit trees on the farm where we work, most of them apples that are used for cider. Sweet potatoes are doing their thing - another six weeks or so and we'll dig up a plant or two to see how they're doing and when we might harvest.
+
+![Corn](images/archive/corn.jpg)
+
+Adrienne is filling in for the farmer's market manager on Bainbridge Island occasionally - she'll be there this Saturday, stop by to say hi and get some veggies from the farmers who are there!
+
+Our farmstand (on Komedal road, just off of Seabold) is open most days. We have Dahlias and greeting cards every day it's open, and often also have eggs, beeswax candles (thanks, bees!), and sometimes spare veggies from the garden. Once bean, corn, and sweet potato harvests are underway, we'll be excited to share those spoils with you as well.

```

Comment:
...on days when it's open, and often also have eggs,...


Once beans, corn, and...